M2/Macaulay2/packages/FastLinAlg.m2

```diff
@@ -77,7 +98,7 @@ StrategyDefault := new HashTable from {
 --protect nanosleep;
 --if (not isGlobalSymbol "nanosleep") then protect getSymbol("nanosleep");
 
-StrategyCurrent = new MutableHashTable from {LexLargest=>0, LexSmallestTerm => 50, LexSmallest=>50, GRevLexSmallestTerm=>0, GRevLexSmallest=>0, GRevLexLargest=>0,Random=>0,RandomNonzero=>0};
```

Comment:
Here you intend to override some values from `StrategyDefault`, so it would be good to write 
```
StrategyCurrent = StrategyDefault ++ {LexLargest=>0, LexSmallestTerm => 50, LexSmallest=>50, GRevLexSmallestTerm=>0, GRevLexSmallest=>0, GRevLexLargest=>0,Random=>0,RandomNonzero=>0};
```
, for you could omit from the list those items with the same values.

Reply:
I think you are looking at the replaced code.  The current code creates a Mutable strategy, based on the default strategy, for the user to play with.  There may be other ways to accomplish this, but this seemed reasonable.